### DISCUSSION TITLE
Algum de vocês conhecem Riot.js ?

### DISCUSSION THREAD

**Discussion by realmocaccino** [frontendbr/forum#734] [Discussão]
Estou montando um styleguide com esse framework.
Ele dispõe de uma abordagem simples e excelente leitura de código.
O Vue.js e React fornecem mais soluções.
Mas o Riot.js está atendendo bem minhas necessidades.

https://github.com/riot/riot

**Comment by expalmer** [comment]
Sim, acho muito legal testar e ver o código de pequenos frames...
[MoonJs](http://moonjs.ga/)
[Choo](https://choo.io/)

**Comment by Cleysonlb** [comment]
Esse MoonJS me lembra muito o VueJS

**Comment by barraponto** [comment]
Eu adorei o Riot. Acho que ele tem muito em comum com o Vue, sendo extremamente simples.
Na real, eu acho que dá pra usar o Vue do mesmo jeito que o Riot -- é só vc escolher usar desse jeito.

**Comment by leandroruel** [comment]
quando Li Riot.js ja pensei em League of Legends... vai entender.

**Comment by bernardodiasc** [comment]
@leandroruel un dos escritórios mais animais que ja tive oportunidade de conhecer https://twitter.com/bernardodiasc/status/709065668369383424

(foi mal, total off topic, pensei a mesma coisa quando li Riot.js)

**Comment by realmocaccino** [comment]
Sim.

Os últimos 4 comentários não enriqueceram em nada o tópico.

**Comment by bernardodiasc** [comment]
@lgustavoms verdade, mas deixa aberta ai. Eu conheço Riot.js, cheguei a ler os docs alguns anos atras e queria muito ter tido tempo pra aprender e usar em algo. Acabei aprendendo Angular, React e outras coisas na frente, mas ainda assim acho que Riot vale a pena tentar. :)

**Comment by didiraja** [comment]
@leandroruel Já eu, li Riot e já pensei que era alguma voltada pra política e manifestacões haha
